report/src/main/webapp/app/layout/toolbar/toolbarTop.view.html

```diff
@@ -83,5 +86,31 @@
     <div class="toolbar-block fontawesome-link" data-toggle="modal" data-target="#helpModal">
       <i class="fas fa-question-circle fa-lg"></i>
     </div>
+    <div class="suite-history-container">
```

Comment:
Maybe it would be good idea to put that into separate modal? Like e.g. `noteModal` (separate controller and view).

Reply:
Done.